src/components/header/Header.jsx

```diff
@@ -0,0 +1,60 @@
+import { React } from "react";
+import { logo } from "../../assets";
+import { Link } from "react-router-dom";
+import "./Header.css";
+const Header = () => {
+	return (
+		<div className="header-container">
+			<header className="header">
+				<div className="header-sub-container">
+					<button className="btn btn-primary btn-icon btn-menu">
+						<i className="fa fa-bars fa-lg" aria-hidden="true"></i>
+					</button>
+					<Link to="/home" className="btn btn-link btn-hide header-link">
+						<img className="img-round logo-container" src={logo} />
```

Comment:
suggestion - in img tag you want to the alt tag I think that's whys netlify deployment failed.

Reply:
Added alt for image.
Also i forgot to run npm run build, will push that in new commit.